src/content/en/2021/privacy.md

```diff
@@ -5,572 +5,541 @@ description: Privacy chapter of the 2021 Web Almanac covering adoption and impac
 
 {{ figure_markup(
   image="nb_websites_using_each_retargeting.png",
   caption="Percentage of pages using a retargeting service",
-  description="Chart showing the most popular services used for retargeting and the number of wesites that use them.",
-  height=421,
+  description="Bar chart showing the most popular services used for retargeting and the number of websites that use them. Google Remarketing Tag is used by 26.92% of desktop sites and 26.64% of mobile sites, Criteo by 1.25% and 1.21% respectively, AdRoll by 0.48% and 0.38%, SharpSpring Ads by 0.12% and 0.09%, Albacross by 0.04% and 0.03%, SteelHouse by 0.03% and 0.02%, Smarter Click by 0.02% and 0.01%, Blue by 0.02% and 0.01%, Cross Pixel by 0.02% and 0.01%, and finally Picreel by 0.01% and 0.01%",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=882622568&format=interactive",
   sheets_gid="1940290757",
-  sql_file="nb_websites_using_each_retargeting.sql"
+  sql_file="nb_websites_using_each_retargeting.sql",
+  width=600,
+  height=580
   )
 }}
 
-
+A number of trackers provide a solution for ad retargeting. The most widely used one, Google Remarketing Tag, is present on 26.92% of websites on desktop and 26.64% of websites on mobile, far and above all other services which are used by less than 1.25% of sites each.
 
 ## How websites handle your sensitive data
 
-Some websites request access to specific features and browser APIs that can impact the user's privacy, for instance by accessing the geolocation data, microphone, camera, etc. While these features usually serve very useful purposes, such as discovering nearby points of interest or allowing people to communicate with each other, there is a risk of exposing sensitive data if the user does not fully understand how those resources are used, or if a site misbehaves. We'll look at how often websites request access to sensitive resources. Moreover, any time a service stores sensitive data, there is the danger of hackers stealing and leaking that data. We'll look at recent data breaches that prove that this danger is real.
+Some websites request access to specific features and browser APIs that can impact the user's privacy, for instance by accessing the geolocation data, microphone, camera, etc. While these features usually serve very useful purposes, such as discovering nearby points of interest or allowing people to communicate with each other, there is a risk of exposing sensitive data if the user does not fully understand how those resources are used, or if a site misbehaves.
 
+{# TODO I feel we should make it clearer that these cannot be activated by user consent (though of course once that consent's been given, it can be abused!). Otherwise is comes across as a bit inaccurate and fear mongering #}
 
```

Comment:
Suggestion: I feel we should make it clearer that these cannot be activated by user consent (though of course once that consent's been given, it can be abused!). Otherwise is comes across as a bit inaccurate and fear mongering

Reply:
see d87c1a4